app/src/main/java/com/ichi2/apisample/helper/AnkiDroidHelper.java

```diff
@@ -323,21 +331,91 @@ public Long addNewDeck(String deckName) {
 
     // @todo: refactor once new version release of "com.ichi2.anki.api" is available
     public String addFileToAnkiMedia(String uriString) {
+        Uri uri = Uri.parse(uriString);
```

Comment:
Not really sure that this extractor should be placed inside AnkiDroidHelper

Reply:
Good point. Factored audio extraction into utility class